lib/log.js

```diff
@@ -35,6 +35,14 @@ function sanitise(raw) {
     raw = sanitiseConfigVariable(raw, 'GITLAB_TOKEN');
   }
 
+  if (config.JIRA_USERNAME) {
```

Comment:
Oooh, I didn't realise we needed to add these to sanitise here!

How about looking for `Object.keys(config)` containing `USERNAME` or `PASSWORD` to catch them automatically?

Reply:
Some creds include `TOKEN` and not `USERNAME`. Agree this is too explicit, but didn't want to focus on this part just yet :)